mods/fire/init.lua

```diff
@@ -43,6 +43,7 @@ end
 -- Flame nodes
 
 minetest.register_node("fire:basic_flame", {
+	description = S("Basic Flame"),
```

Comment:
I think just 'Flame' is a better description.
Perhaps both nodes should be described as 'Fire' instead?

Reply:
The characteristics of the two kinds of flames are different.for basic flame, I think it's OK to change it to "fire".